dask/dataframe/tests/test_multi.py

```diff
@@ -2520,6 +2520,43 @@ def test_concat_ignore_order(ordered):
     assert_eq(result, expected)
 
 
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        "Int64",
+        "Float64",
+        pytest.param(
+            "int64[pyarrow]",
+            marks=pytest.mark.skipif(
+                not PANDAS_GT_150, reason="Support for ArrowDtypes added in pandas 1.5"
+            ),
+        ),
+        pytest.param(
+            "float64[pyarrow]",
+            marks=pytest.mark.skipif(
+                not PANDAS_GT_150, reason="Support for ArrowDtypes added in pandas 1.5"
+            ),
+        ),
```

Comment:
We also want to skip if `pyarrow` isn't installed (sorry for not catching that earlier)

Reply:
Can make this change, but seems like the test suite is passing without it - not sure if there's a case I'm missing where we would get to this point without pyarrow

Reply:
Fair point. We don't currently test with `pandas < 1.5` and without `pyarrow`, which is why CI is passing. Though using `pyarrow`-backed dtypes in `pandas` does require `pyarrow` to be installed. I could see this possibly showing up in the future if we, for example, increase our minimum supported `pandas` version to 1.5 (the dataframe mindeps builds would start failing since it doesn't include `pyarrow`)